mockgen/internal/tests/generics/other/other.go

```diff
@@ -9,3 +9,13 @@ type Three struct{}
 type Four struct{}
 
 type Five interface{}
+
+type Twenty[R, S, T any, Z any] interface {
+	Twenty(S, R) (T, Z)
+}
+
+type TwentyThree[U, V any] interface {
```

Comment:
naming: Could the more sophisticated types be named by their function instead of a number? It took some time for me to organize which cases were tested while reviewing.